tests/Execution/string-literals.java

```diff
@@ -0,0 +1,23 @@
+// RUN: javac %s -d %t
```

Comment:
I think you need to add `-encoding utf-8` to the arguments of `javac` for it to allow UTF-8 in the source file.
Not sure why, and not sure why not on mac?!